controllers/clusterdevfileregistrieslist_controller.go

```diff
@@ -68,10 +76,43 @@ func (r *ClusterDevfileRegistriesListReconciler) Reconcile(ctx context.Context,
 		return ctrl.Result{}, err
 	}
 
+	if clusterDevfileRegistriesList.Status.Conditions == nil || len(clusterDevfileRegistriesList.Status.Conditions) == 0 {
+		meta.SetStatusCondition(&clusterDevfileRegistriesList.Status.Conditions, metav1.Condition{
+			Type:    typeAvailableClusterDevfileRegistriesList,
+			Status:  metav1.ConditionUnknown,
+			Reason:  "Reconciling",
+			Message: "Starting reconciliation",
```

Comment:
It might be better to use a different word for the reason than "Reconciling"?

Reply:
@johnmcollier "Updating available devfile registry cluster list"?

Reply:
That's more so for the message. Reason you generally want one or two words. `NotReady`? Or something thereof?

Reply:
"Ready"/"NotReady"?

Reply:
Addressed in https://github.com/devfile/registry-operator/pull/49/commits/33593bdd7b4a2b25c7bc3f5feafa69109841c45f